.github/global.yml

```diff
@@ -41,6 +41,9 @@
   - name: dependencies
     description: '[bot] PRs that update a dependency file'
     color: '#0366d6'
+  - name: profane content 🤬
+    description: 'Either the title or body text contains profanity'
+    color: 'FBCA04'
```

Comment:
The label is required (if not previously implemented a new one gets created), please see [Label requirements section of the profanity filter's README](https://github.com/IEvangelist/profanity-filter?tab=readme-ov-file#%EF%B8%8F-label-requirements)